react-native/react/nav.ios.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-import React, { Component, TabBarIOS, View, Navigator, Text, TouchableOpacity, StyleSheet } from 'react-native'
+import React, { Component, TabBarIOS, View, Navigator, Text, TouchableOpacity, StyleSheet, NativeAppEventEmitter } from 'react-native'
```

Comment:
I'm not convinced that `Nav` is the right place for this to live. Time travel isn't really part of it (or any component), and I was actually planning to wrap nav in another component (which can display modals) as part of an upcoming PR — at that point it wouldn't even be the top-level component doing this, it would just be some random component that happens to exist most of the time.

You might be able to come up with something better than this, but what about just having a `setupTimeTravel(store)` call in `index.js`, right after configureStore, which binds the listeners?